app/helpers/spree/admin/orders_helper.rb

```diff
@@ -3,10 +3,10 @@
 module Spree
   module Admin
     module OrdersHelper
-      def event_links
+      def event_links(order)
```

Comment:
You're right, a helper shouldn't have an instance variable, this is better.

Another option would be to move this to a class, which could then be instantiated with a reference to the order. But I'm not sure that's any cleaner.

Reply:
> Another option would be to move this to a class, which could then be instantiated with a reference to the order. But I'm not sure that's any cleaner.

I think this is a cleaner way given the usage of order in this helper. An extra class, but private methods here could have been moved to that class as well. 👍🏻